### PR TITLE
Update release_process with time line guidance

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

**Comment by pshipton** [comment]
Hide whitespace changes to view.

File: doc/processes/release_process.md [hunk 1]
```diff
@@ -57,23 +57,30 @@ data.
...
+cannot be met, a new estimate will be publicly announced after analysis of
+the issues blocking the release. This is not meant to be an exact date but
+to provide a new maximum, such as "provide a source code release within the
+next two months".
+
 ## Quality bar
-Work with the AdoptOpenJDK community to define a common quality bar for 
-any OpenJDK-based JDK.  The aim is to define a set of tests, both 
-functional and stress, and 3rd party applications that should pass for 
+Work with the AdoptOpenJDK community to define a common quality bar for
```
Comment (DanHeidinga) [inline doc/processes/release_process.md:74]:
```suggestion
Work with the Adoptium community to define a common quality bar for
```

File: doc/processes/release_process.md [hunk 2]
```diff
@@ -57,23 +57,30 @@ data.
...
 every release.
 
 ## Platform support
-Define the platforms that are supported by a release.  The set of 
-platforms will depend on the availability of hardware at OpenJ9 and 
-AdoptOpenJDK that can be used to validate the platform meets the quality 
-bar.  The code base may support more platforms than can be validated by 
+Define the platforms that are supported by a release.  The set of
+platforms will depend on the availability of hardware at OpenJ9 and
+AdoptOpenJDK that can be used to validate the platform meets the quality
```
Comment (DanHeidinga) [inline doc/processes/release_process.md:82]:
```suggestion
Adoptium that can be used to validate the platform meets the quality
```

File: doc/processes/release_process.md [hunk 3]
```diff
@@ -83,14 +90,14 @@ that will be tracked for the release.  These measures should include the
...
+need to update / modify the architecture.  Some regressions may simply be
+found too late in a release to address them.
+The project may decide to trade performance - whether throughput, startup
+or fooprint - for another important measure.
 Any regressions should be documented as part of the release plan.
 
-Note, detecting regressions will be difficult in the open project 
-given the limited hardware resources available.  Work is being done at 
+Note, detecting regressions will be difficult in the open project
+given the limited hardware resources available.  Work is being done at
```
Comment (DanHeidinga) [inline doc/processes/release_process.md:100]: AdoptOpenJDK->Adoptium on the next line as well

**Review by DanHeidinga — APPROVED** [review]
New paragraph looks good.  We should do a global search replace to update AdoptOpenJDK to Adoptium

**Review by DanHeidinga — APPROVED** [review]
lgtm

**Comment by pshipton** [comment]
@mstoodle for your review

File: doc/processes/release_process.md [hunk 4]
```diff
@@ -32,49 +32,57 @@ and one has an external test repo:
...
-support (LTS) release cadence.  OpenJDK 8 is released quarterly and 
-other LTS releases will likely follow similar release cadences.  There 
-should be monthly milestone builds as the project builds towards the 
+OpenJ9 should aim to release quarterly to match the OpenJDK long term
+support (LTS) release cadence.  OpenJDK 8, 11, 17 are released quarterly and
+other LTS releases will likely follow similar release cadences.  There
+should be milestone builds created as the project advances towards the
 next release.
 
+OpenJ9 will endeavor to finalize a source code release within two weeks of
```
Comment (mstoodle) [inline doc/processes/release_process.md:67]:
```suggestion
OpenJ9 will endeavour to finalize a source code release within two weeks of
```

**Comment by mstoodle** [comment]
one typo, though it's really a nit. otherwise looks good to me too; thanks, Peter!

**Review by mstoodle — APPROVED** [review]

File: doc/processes/release_process.md (hunk 4 above)
Comment (pshipton) [reply doc/processes/release_process.md]: Updated. Pls merge the PR is all is good now.

**Pull request merged**